darwin/types.py

```diff
@@ -0,0 +1,6 @@
+from typing import Dict, List
+
+Point = Dict[str, float]
+BoundingBox = Dict[str, float]
+Polygon = List[Point]
+ComplexPolygon = List[Polygon]
```

Comment:
Do you like something like this? I used it in `darwin/torch/utils.py`, and to me it looks a bit better than `Union[List[Point], List[List[Point]]`

Reply:
Unfortunately, `TypedDict` would have been better, but it's only available in `typing` since Python 3.8

Reply:
I like it, I feel however that types.py and datatypes.py should be merged and more interlinked